backend/src/passwords.cpp

```diff
@@ -0,0 +1,72 @@
+#include <iostream>
+#include "passwords.hpp"
+
+namespace passwords
```

Comment:
**suggestion**: Change namespace (and file) name to `password`, just like `database`.